test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -1212,6 +1212,13 @@ def _validate_argument_spec(self, docs, spec, kwargs, context=None):
                     code='parameter-alias-repeated',
                     msg=msg
                 )
+            if not context and arg == 'state':
+                bad_states = {'list', 'info'} & set(data.get('choices', set()))
```

Comment:
```suggestion
                bad_states = {'list', 'info'} & set(data.get('choices', set()))
```
Potentially `get` could also be a bad state.

Reply:
@felixfontein weird... the diff looks identical, what have you changed?

Reply:
I think it's okay to not validate bad get. But yes, it could be even some invalid non-iterable data struct which would explode.

Reply:
It could be that I accidentally pressed the "suggestion" button. I only wanted to mention that I think `get` maybe also should be in this list.

Reply:
Ah. I thought you're talking about `dict.get()` being not enough for the validator.

Reply:
I misunderstood that too :)

The trouble with 'get' is that I could imagine it being used in the 'download'/'fetch' context rather than the tell-me-about-it context.

Reply:
I wonder if there are more correct than wrong uses of `get` :) I mainly know its use in `route53`, where it shouldn't be a state.

Reply:
3 examples today.

1. route53
2. gconftool2
3. xfconf

All 3 are candidates for _info